### DISCUSSION TITLE
setup sponsor

### DISCUSSION THREAD

**Discussion by MallikarjunHt** [EddieHubCommunity/support#531] [Ideas]
Hi can you make a video on how to setup `GitHub  Sponsor` and what are the options and how to add. 
Thank You

**Comment by eddiejaoude** [comment]
hey @MallikarjunHt , thank you so much for sponsoring me on GitHub 👍 (btw are you in our Discord and if yes what is your username so I can give you the `sponsor` role 🤓 )

I created this view fairly recently, have you seen it? Is there anything missing that you would like to see? https://www.youtube.com/watch?v=l8WCmx2G5bI
